python/django_bolt/serializers/decorators.py

```
@@ -41,6 +41,38 @@ def __call__(self, serializer: Serializer, /) -> Serializer: ...
+    """Refuse a classmethod/staticmethod model validator instead of ignoring it.
+
+    A model validator receives the constructed instance, so neither descriptor
+    can express one. Silently collecting nothing would leave a serializer that
+    looks validated and is not.
+    """
+    if isinstance(func, (classmethod, staticmethod)):
+        kind = type(func).__name__
+        raise TypeError(
+            f"@model_validator cannot wrap a {kind}: a model validator receives the "
+            f"constructed instance, not the class. Drop the @{kind} decorator."
+        )
```

Ruff (0.16.1)

[warning] Avoid specifying long messages outside the exception class

(TRY003)